src/ch/blinkenlights/android/vanilla/PlaybackService.java

```diff
@@ -2104,6 +2104,22 @@ public Notification createNotification(Song song, int state, int mode)
 		}
 		if(Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
 			notification.visibility = Notification.VISIBILITY_PUBLIC;
+
+			RemoteViews viewsPublic = new RemoteViews(getPackageName(), R.layout.notification);
+			viewsPublic.setInt(R.id.title, "setText", R.string.app_name);
```

Comment:
Ah, i missed that setTextView() only takes a char (and not a res-id), so this this looks fine then.